app/controllers/registrations_controller.rb

```diff
@@ -14,8 +14,15 @@ def create
     ensure_admin || return
     build_resource(sign_up_params)
     resource.save
+    success = resource.save
     yield resource if block_given?
-    redirect_to users_path
+    if success
+      redirect_to users_path,
+        notice:'New User Created'
+    else
+      redirect_to users_path,
+        alert: 'User name already exist!'
```

Comment:
Technically not necessarily true. Can instead just use `resource.errors.full_messages.join(', ')`